pennylane/operation.py

```diff
@@ -883,7 +883,7 @@ def __init__(self, *params, wires=None, do_queue=True, id=None):
 
         self._check_batching(params)
 
-        self.data = list(params)  #: list[Any]: parameters of the operator
+        self.data = [np.array(p) if isinstance(p, list) else p for p in params]
```

Comment:
Could there be a scenario where a parameter is a list of , say, JAX arrays, and the conversion to vanilla numpy would cause a problem? Or should this not happen anyway? :thinking:

Reply:
Good question! I honestly don't know if there is any case in PL where a parameter is a list. The only case I found is the `K_list` parameter in `QubitChannel`. Maybe it's worth checking this

Reply:
If there were special cases that actually wanted lists, they can override the initialization function to not perform this step.

Reply:
That's a good point. It might duplicate the step of creating `self.data` if `super().__init__` is used, but that should not be a problem, in my opinion.